charts/timescaledb-single/values.yaml

```
@@ -261,6 +261,7 @@ patroni:
     pg_hba:
     - local     all             postgres                              peer
     - local     all             all                                   md5
+    # replace reject with md5 for patroni.bootstrap.dcs.postgresql.parameters.ssl: off
```

```suggestion
    # Replace `reject` with `md5` to allow connections when SSL
    # is disabled with `patroni.bootstrap.dcs.postgresql.parameters.ssl: off`.
```

Tried to make it a bit clearer.